Update link to Kelsey's repo

gcdf opened the pull request:
Kelsey made some changes to her slides and we added a README that links to them. Thanks!

kaymmm commented:
as per my message in the forums, please change the info for workshops in the google spreadsheet (not linked here for privacy reasons). there are 2 sheets, and the taglines should be entered in the second sheet. this page is generated programmatically using the data in that spreadsheet.

jojokarlin committed:
Update link to Kelsey's repo

Kelsey made some changes to her slides and we added a README that links to them. Thanks!
